### PR TITLE
Return Account from create_account(), add get_metadata()

### LINKED ISSUE #743: Python return `Account` from `wallet.create_account()`

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Return Account from create_account(), add get_metadata()

## Links to any relevant issues

Fixes #743 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Running the modified examples

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/examples/how_tos/accounts_and_addresses/create_account.py [hunk 1]
```diff
@@ -28,8 +28,7 @@
     raise Exception(".env MNEMONIC is undefined, see .env.example")
 
 # Store the mnemonic in the Stronghold snapshot, this only needs to be done once
-wallet.store_mnemonic(
-    os.environ['MNEMONIC'])
+# wallet.store_mnemonic(os.environ['MNEMONIC'])
```
Comment (thibault-martinez) [inline bindings/python/examples/how_tos/accounts_and_addresses/create_account.py]: 🤔

File: bindings/python/iota_sdk/wallet/account.py [hunk 2]
```diff
@@ -46,6 +43,12 @@ def _call_account_method(self, method, data=None):
 
         return message
 
+    def get_metadata(self) -> AccountMetadata:
+        """
+        A generic `prepare_burn()` function that can be used to prepare the burn of native tokens, nfts, foundries and aliases.
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/wallet/account.py]: :carlos:

File: bindings/python/examples/how_tos/accounts_and_addresses/create_account.py (hunk 1 above)
Comment (Thoralf-M) [reply bindings/python/examples/how_tos/accounts_and_addresses/create_account.py]: Uncommented again
Disabled it during testing so I don't have to clean the db or create another example, but maybe we should split wallet and account creation

File: bindings/python/iota_sdk/wallet/account.py (hunk 2 above)
Comment (Thoralf-M) [reply bindings/python/iota_sdk/wallet/account.py]: Thanks :trollface:

**Review by thibault-martinez — APPROVED** [review]

File: bindings/python/iota_sdk/wallet/account.py [hunk 3]
```diff
@@ -46,6 +43,12 @@ def _call_account_method(self, method, data=None):
 
         return message
 
+    def get_metadata(self) -> AccountMetadata:
+        """
+        Get the accounts metadata.
+        """
+        return AccountMetadata(self.meta["alias"], self.meta["coinType"], self.meta["index"])
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/wallet/account.py:50]: Shouldn't we parse this only once when creating it ?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/wallet/account.py:50]: Don't think that this really makes a difference
In nodejs we did it also like this, I can change in both if you want

Comment (thibault-martinez) [reply bindings/python/iota_sdk/wallet/account.py:50]: Was just curious, all good 👍🏻

File: bindings/python/examples/wallet/logger.py [hunk 4]
```diff
@@ -39,4 +39,4 @@
 account = wallet.store_mnemonic(os.environ["MNEMONIC"])
 
 account = wallet.create_account('Alice')
-print(json.dumps(account, indent=4))
+print(account.get_metadata())
```
Comment (kwek20) [inline bindings/python/examples/wallet/logger.py:42]: this doenst need a dumps still ?

**Review by DaughterOfMars — APPROVED** [review]

File: bindings/python/examples/wallet/logger.py (hunk 4 above)
Comment (Thoralf-M) [reply bindings/python/examples/wallet/logger.py:42]: No

**Review by kwek20 — APPROVED** [review]

**Pull request merged**